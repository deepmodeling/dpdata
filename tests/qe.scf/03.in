&control
  calculation='scf'
  prefix='water'
  pseudo_dir='../pseudo'
  outdir='./out'
  restart_mode='from_scratch'
  nstep=20000
  disk_io='none'
  max_seconds=10000
  tprnfor=.true.
  tstress=.false.
/
&system
  ibrav=1
  a=10
  nat=5
  ntyp=2,
  ecutwfc=110
  input_dft='PBE'
/
&electrons
  electron_maxstep = 1000
  mixing_beta = 0.5
/
&ions
/
&cell
/
ATOMIC_SPECIES
H   1.0  H.pbe-van_ak.UPF
C   1.0  C.pbe-van_ak.UPF 
ATOMIC_POSITIONS {angstrom}
H 5.41646 4.01132 3.51118
H 4.13161 4.70677 4.43116
H 5.63096 5.52167 4.45038
H 5.49988 4.00341 5.34265
C 5.19271 4.55775 4.43687
