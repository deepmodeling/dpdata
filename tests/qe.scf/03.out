
     Program PWSCF v.7.1 starts on  5Nov2023 at 13:51:57 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
         "P. Giannozzi et al., J. Chem. Phys. 152 154105 (2020);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on    28 processors

     MPI processes distributed on     1 nodes
     59927 MiB available memory on the printing compute node when the environment starts
 
     Waiting for input...
     Reading input from standard input
Warning: card &CELL ignored
Warning: card / ignored

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  4

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want


     gamma-point specific algorithms are used
 
     R & G space division:  proc/nbgrp/npool/nimage =      28
     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         446     446    110                37558    37558    4694
     Max         448     448    112                37564    37564    4708
     Sum       12517   12517   3125              1051747  1051747  131683
 
     Using Slab Decomposition
 


     bravais-lattice index     =            1
     lattice parameter (alat)  =      18.8973  a.u.
     unit-cell volume          =    6748.3345 (a.u.)^3
     number of atoms/cell      =            5
     number of atomic types    =            2
     number of electrons       =         8.00
     number of Kohn-Sham states=            4
     kinetic-energy cutoff     =     110.0000  Ry
     charge density cutoff     =     440.0000  Ry
     scf convergence threshold =      1.0E-06
     mixing beta               =       0.5000
     number of iterations used =            8  plain     mixing
     Exchange-correlation= PBE
                           (   1   4   3   4   0   0   0)

     celldm(1)=  18.897261  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for H  read from file:
     ../pseudo/H.pbe-van_ak.UPF
     MD5 check sum: a537b41238c433fb4068aaff04c6fb4a
     Pseudo is Ultrasoft, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  615 points,  1 beta functions with: 
                l(1) =   0
     Q(r) pseudized with  8 coefficients,  rinner =    0.800

     PseudoPot. # 2 for C  read from file:
     ../pseudo/C.pbe-van_ak.UPF
     MD5 check sum: ecc3b1e1c1ebe006fc976faab44e17a8
     Pseudo is Ultrasoft, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  721 points,  4 beta functions with: 
                l(1) =   0
                l(2) =   0
                l(3) =   1
                l(4) =   1
     Q(r) pseudized with  8 coefficients,  rinner =    0.800   0.800   0.800


     atomic species   valence    mass     pseudopotential
        H              1.00     1.00000     H ( 1.00)
        C              4.00     1.00000     C ( 1.00)

     No symmetry found



   Cartesian axes

     site n.     atom                  positions (alat units)
         1           H   tau(   1) = (   0.5416460   0.4011320   0.3511180  )
         2           H   tau(   2) = (   0.4131610   0.4706770   0.4431160  )
         3           H   tau(   3) = (   0.5630960   0.5521670   0.4450380  )
         4           H   tau(   4) = (   0.5499880   0.4003410   0.5342650  )
         5           C   tau(   5) = (   0.5192710   0.4557750   0.4436870  )

     number of k points=     1
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     Dense  grid:   525874 G-vectors     FFT dimensions: ( 128, 128, 128)

     Estimated max dynamical RAM per process >      30.88 MB

     Estimated total dynamical RAM >     864.60 MB

     Initial potential from superposition of free atoms

     starting charge       7.9999, renormalised to       8.0000

     negative rho (up, down):  1.567E-03 0.000E+00
     Starting wfcs are    8 randomized atomic wfcs

     total cpu time spent up to now is        0.7 secs

     Self-consistent Calculation

     iteration #  1     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  2.0

     negative rho (up, down):  3.779E-03 0.000E+00

     total cpu time spent up to now is        0.9 secs

     total energy              =     -16.02157560 Ry
     estimated scf accuracy    <       0.70040947 Ry

     iteration #  2     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  8.76E-03,  avg # of iterations =  2.0

     negative rho (up, down):  4.476E-03 0.000E+00

     total cpu time spent up to now is        1.1 secs

     total energy              =     -16.15192549 Ry
     estimated scf accuracy    <       0.04051988 Ry

     iteration #  3     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  5.06E-04,  avg # of iterations =  1.0

     negative rho (up, down):  5.628E-03 0.000E+00

     total cpu time spent up to now is        1.2 secs

     total energy              =     -16.14738168 Ry
     estimated scf accuracy    <       0.01107922 Ry

     iteration #  4     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  1.38E-04,  avg # of iterations =  2.0

     negative rho (up, down):  5.562E-03 0.000E+00

     total cpu time spent up to now is        1.4 secs

     total energy              =     -16.14875721 Ry
     estimated scf accuracy    <       0.00199039 Ry

     iteration #  5     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  2.49E-05,  avg # of iterations =  2.0

     negative rho (up, down):  5.643E-03 0.000E+00

     total cpu time spent up to now is        1.5 secs

     total energy              =     -16.14865735 Ry
     estimated scf accuracy    <       0.00032165 Ry

     iteration #  6     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  4.02E-06,  avg # of iterations =  3.0

     negative rho (up, down):  5.623E-03 0.000E+00

     total cpu time spent up to now is        1.7 secs

     total energy              =     -16.14878943 Ry
     estimated scf accuracy    <       0.00003385 Ry

     iteration #  7     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  4.23E-07,  avg # of iterations =  2.0

     negative rho (up, down):  5.612E-03 0.000E+00

     total cpu time spent up to now is        1.9 secs

     total energy              =     -16.14877906 Ry
     estimated scf accuracy    <       0.00003129 Ry

     iteration #  8     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  3.91E-07,  avg # of iterations =  2.0

     negative rho (up, down):  5.611E-03 0.000E+00

     total cpu time spent up to now is        2.0 secs

     total energy              =     -16.14878120 Ry
     estimated scf accuracy    <       0.00000208 Ry

     iteration #  9     ecut=   110.00 Ry     beta= 0.50
     Davidson diagonalization with overlap
     ethr =  2.61E-08,  avg # of iterations =  3.0

     negative rho (up, down):  5.614E-03 0.000E+00

     total cpu time spent up to now is        2.2 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 ( 65842 PWs)   bands (ev):

   -16.9477  -9.4968  -9.3276  -9.0865

     highest occupied level (ev):    -9.0865

!    total energy              =     -16.14878191 Ry
     estimated scf accuracy    <       0.00000062 Ry

     The total energy is the sum of the following terms:
     one-electron contribution =     -40.61130475 Ry
     hartree contribution      =      21.13447342 Ry
     xc contribution           =      -6.37239920 Ry
     ewald contribution        =       9.70044862 Ry

     convergence has been achieved in   9 iterations

     Forces acting on atoms (cartesian axes, Ry/au):

     atom    1 type  1   force =     0.01321833    0.00035039    0.00596821
     atom    2 type  1   force =    -0.03792494    0.00068257    0.00934494
     atom    3 type  1   force =     0.03068281    0.04387976   -0.00506668
     atom    4 type  1   force =    -0.00384202    0.00079975   -0.01397316
     atom    5 type  2   force =    -0.00213417   -0.04571247    0.00372669

     Total force =     0.083354     Total SCF correction =     0.000416
 
     init_run     :      0.39s CPU      0.59s WALL (       1 calls)
     electrons    :      1.44s CPU      1.49s WALL (       1 calls)
     forces       :      0.09s CPU      0.09s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.02s CPU      0.05s WALL (       1 calls)
     potinit      :      0.09s CPU      0.13s WALL (       1 calls)
     hinit0       :      0.05s CPU      0.10s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.22s CPU      0.24s WALL (       9 calls)
     sum_band     :      0.30s CPU      0.32s WALL (       9 calls)
     v_of_rho     :      0.60s CPU      0.65s WALL (      10 calls)
     newd         :      0.24s CPU      0.29s WALL (      10 calls)
     mix_rho      :      0.12s CPU      0.12s WALL (       9 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (      19 calls)
     init_us_2:cp :      0.01s CPU      0.01s WALL (      19 calls)
     regterg      :      0.21s CPU      0.22s WALL (       9 calls)

     Called by *egterg:
     rdiaghg      :      0.00s CPU      0.02s WALL (      28 calls)
     h_psi        :      0.20s CPU      0.22s WALL (      29 calls)
     s_psi        :      0.00s CPU      0.00s WALL (      29 calls)
     g_psi        :      0.00s CPU      0.00s WALL (      19 calls)

     Called by h_psi:
     h_psi:calbec :      0.01s CPU      0.02s WALL (      29 calls)
     vloc_psi     :      0.19s CPU      0.19s WALL (      29 calls)
     add_vuspsi   :      0.00s CPU      0.01s WALL (      29 calls)

     General routines
     calbec       :      0.01s CPU      0.02s WALL (      42 calls)
     fft          :      0.38s CPU      0.45s WALL (      92 calls)
     ffts         :      0.03s CPU      0.03s WALL (       9 calls)
     fftw         :      0.17s CPU      0.18s WALL (     128 calls)
 
     Parallel routines
 
     PWSCF        :      1.98s CPU      2.31s WALL

 
   This run was terminated on:  13:52: 0   5Nov2023            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
